### ISSUE TITLE
gitignore

### ISSUE THREAD

**Issue by amosmos** [spatie/laravel-medialibrary#350]
Hi,

Thank you for such a great package!!

After developing and testing our app for few weeks, we were ready to launch, and only then we've seen that the media directory is not ignored...

I've seen the issue that made you remove the gitignore file creation - it was due to this process being too long when using cloud storage.

However I do believe that when using a local storage, gitignore is super important. So how about doing that only for local storage (as was suggested in the issue conversation)? Or at least a big notice about this in the docs (which still say "The package will add a .gitignore file to the directory where the media will be stored").

Bottom line, right now should I just manually add that file?

Thanks,
Amos

**Comment by freekmurze** [comment]
You should manually ignore the directory where the media is stored. I'll put a little note on this in the docs.

**Comment by snapey** [comment]
I see "_Don't forget to ignore the directory of your configured disk so the files won't end up in your git repo._" in the documentation, but medialibrary also creates a temp folder in storage/medialibrary and after some crashes uploading images larger than Intervention could cope with I was left with quite a few directories in temp and also committed to Git.

Could the creation of the storage/medialibrary folder also create a .gitignore file?
